backtester/config/strategyexamples/dca-api-candles.strat

```diff
@@ -11,7 +11,7 @@
  },
  "currency-settings": [
   {
-   "exchange-name": "binance",
+   "exchange-name": "bybit",
```

Comment:
Please update `TestGenerateConfigForDCAAPICandlesMultipleCurrencies` to point to bybit or this will be overwritten